webrender/res/ps_image.fs.glsl

```diff
@@ -4,6 +4,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#ifdef WR_YUV_IMAGE
+uniform sampler2D sYTexture;
+// TODO: investigate storing the U and V planes in the same texture.
+uniform sampler2D sUTexture;
```

Comment:
@glennw do you think we can afford allocating 2 more slots for the UV stuff? Doesn't look like they are going to be used for anything else, and we are limited to 16 slots in total.